README.md

```diff
@@ -215,7 +215,7 @@ command won't stop it, but will initialize its data directory and starts it if
 it's not already running.
 
 ```
-$ pgenv use 10.4
+pgenv use 10.4
```

Comment:
Removing the `$` makes it harder to distinguish what's input and what's output. It's a handy affordance. What's the reason to remove it?

Reply:
I thought you wanted to remove the `$`, my fault, I will revert it back.  For example, in the installation section there is no `$` before the `git` command or other commands. Do we place it all over, or remove from everywhere?

Reply:
My rule of thumb has been:

*   If the block shows inputs (what you type in) and outputs (the result of tying it in), use a `$` at the start of the input lines to distinguish them from the output lines. Specify no formatting. Example:

    ```
    $ echo hello there
    hello there
    ```

*   If the block is all inputs, just pure shell programming syntax, like what would be in a shell script, use no `$` and a "sh" fenced block. Example:

    ``` sh
    if [ "$1" == "yes" ]; then
        echo hello there
    fi
    ```

Reply:
Fix in beb5f98